lib/rules/declaration-property-value-no-unknown/index.mjs

```diff
@@ -24,6 +24,8 @@ const messages = ruleMessages(ruleName, {
 	rejected: (property, value) => `Unexpected unknown value "${value}" for property "${property}"`,
 	rejectedParseError: (property, value) =>
 		`Cannot parse property value "${value}" for property "${property}"`,
+	rejectedValueFunction: (property, valueFunction) =>
```

Comment:
If we add a new message type it should be scoped.
i.e. the function name should be present
e.g. `Unexpected unknown value "${value}" for function "${name}"`

Either change the message or simple reuse `rejected`.

Reply:
Let's pass it whole to the existing `rejected` function, e.g.:

'Unexpected unknown value "rgb(255, 255)" for property "color"'

That's clear enough, and an improvement on what we currently have.